tests/integration/release/test_release.py

```diff
@@ -60,11 +61,12 @@ def test_unnoised_id_cols(dataset_name: str, request: FixtureRequest) -> None:
     original = initialize_dataset_with_sample(dataset_name)
     noised_data = request.getfixturevalue("noised_data")
     check_noised, check_original, _ = _get_common_datasets(original, noised_data)
-    assert (
-        (
-            check_original.reset_index()[unnoised_id_cols]
-            == check_noised.reset_index()[unnoised_id_cols]
+    with check:
```

Comment:
There's only one assertion, so what's the point of putting this in a `check` block?

Reply:
Oh yeah this is actually unnecessary